### PR TITLE
DTSPO-17136-apim-upgrade

### PR DESCRIPTION
**Pull request by JoshuaGlassKainos**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSPO-17136

### Change description ###
added trigger_migration var and trigger_migration_temp_pip as this requires further time before we can destory this resource
these triggers tell the apim service to upgrade from stv1 to stv2

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### components/apim/main.tf
- Removed the module \"ctags\" block and its references.
- Added the \"trigger_migration\" and \"trigger_migration_temp_pip\" variables and their respective values for the \"api-mgmt\" module.

### environments/sbox/sbox.tfvars
- Added a new variable \"trigger_migration_temp_pip\" with a value of true.

### environments/variables.tf
- Added the \"trigger_migration\" and \"trigger_migration_temp_pip\" variables with default values of false.

### CODE REVIEW

File: environments/test/test.tfvars [hunk 1]
```diff
@@ -3,6 +3,7 @@ location               = "uksouth"
 env                    = "perftest"
 subscription           = "test"
 certificate_name_check = false
+trigger_migration      = false
```
Comment (thomast1906) [inline environments/test/test.tfvars:6]: Why add this if the default is false?

Comment (JoshuaGlassKainos) [reply environments/test/test.tfvars:6]: added in as this was going to be used for future usage, I can remove if you would like.

Comment (JoshuaGlassKainos) [reply environments/test/test.tfvars]: removed from tfvars as been defaulted to false as highlighted

**Review by MartyFox — APPROVED** [review]

File: environments/sbox/sbox.tfvars [hunk 2]
```diff
@@ -10,6 +10,8 @@ data_subscription              = "bf308a5c-0624-4334-8ff8-8dca9fd43783"
 key_vault_subscription         = "b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb"
 privatedns_subscription        = "1497c3d7-ab6d-4bb7-8a10-b51d03189ee3"
 oms_env                        = "sandbox"
+trigger_migration              = false
```
Comment (thomast1906) [inline environments/sbox/sbox.tfvars:13]: remove? default is false

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**